### PR TITLE
playground: print version when filling binpath

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve? 
When filling in the binpath, it isbetter to print out the instance using binpath

### What is changed and how it works?

before
```
$ tiup playground v7.5.0 --pd.binpath /Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:v7.5.0
Start tikv instance:v7.5.0
Start tidb instance:v7.5.0
```

after
```
$ ./tiup-playground v8.1.0 --pd.mode ms --pd 1 --tiflash 0 --scheduling 3 --tso 2 --pd.binpath /Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:/Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:/Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:/Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:/Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:/Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start pd instance:/Users/pingcap/CS/PingCAP/pd/bin/pd-server
Start tikv instance:v8.1.0
Start tidb instance:v8.1.0
```

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Release notes:

```release-note
print version when filling binpath
```

### CODE REVIEW

**Comment by HuSharp** [comment]
cc @lhy1024

**Review by lhy1024 — APPROVED** [review]

File: components/playground/instance/instance.go [hunk 1]
```diff
@@ -70,6 +70,8 @@ type Instance interface {
 	// Wait Should only call this if the instance is started successfully.
 	// The implementation should be safe to call Wait multi times.
 	Wait() error
+	// BinPathCheck return the bin path to check component version.
+	BinPathCheck(utils.Version) string
```
Comment (breezewish) [inline components/playground/instance/instance.go:74]: How about simply return the bin path (and call it `BinPath`)? The current API mixes two different things together, which is confusing and not useful for other purposes.

Comment (HuSharp) [reply components/playground/instance/instance.go]: How about advancing the `PrepareBinary` function?

**Review by nolouch — APPROVED** [review]
lgtm

**Comment by HuSharp** [comment]
PTAL, thx! @xhebox

**Review by xhebox — APPROVED** [review]

**Pull request merged**